### ISSUE TITLE
Can't use arguments for module inspection

### ISSUE THREAD

**Issue by jreisserVeoL** [antonbabenko/pre-commit-terraform#76]
Hello! 

I'm not sure why but I can't add arguments to `terraform_tflint` hook. I'm trying to use `tflint --module` or any argument but I always get the following error message:

`getopt: unrecognize option '--module'`

## Project structures
I'm using official terraform modules which are located in the `.terraform` file and custom modules.
```
.
├── .terraform
├── .pre-commit-config.yaml
├── main.tf
├── variables.tf
└── outputs.tf
```
or

```
.
├── custom_module
│   ├── main.tf
│   ├── outputs.tf
│   └── variables.tf
│
├── .terraform
├── .pre-commit-config.yaml
├── main.tf
├── outputs.tf
└── variables.tf
 ```

`pre-commit-config.yaml` just contains the following hooks : 

```yaml
- repo: git://github.com/antonbabenko/pre-commit-terraform
  rev: v1.21.0
  hooks:
    - id: terraform_tflint
      args: ['--module']
```
I use `TFLint version 0.13.1`, `pre-commit 1.20.0` and `pre-commit-terraform v1.21.0`. pre-commit is up to date.

Is there something I'm missing ? If you have any suggestion or workaround to help us with this problem, I'd greatly appreciate it.

**Comment by grzegorzlisowski** [comment]
To make it work you need two things:

1. Following PR merged into master and new version released (or have your own fork): https://github.com/antonbabenko/pre-commit-terraform/pull/77

2. One of the latest [PRs](https://github.com/antonbabenko/pre-commit-terraform/pull/75) merged to the mainline changed the way of passing arguments to the tflint. 

I have no clue how this is working for anyone: https://github.com/antonbabenko/pre-commit-terraform#notes-about-terraform_tflint-hooks

```hooks:
  - id: terraform_tflint
    args: ['--deep']```

Maybe I'm missing something important. Per the code in the the script and according to the way how pre-commit passes arguments to the scripts (https://pre-commit.com/#passing-arguments-to-hooks), the only solution which I see is this:

```hooks:
  - id: terraform_tflint
    args: ['--args', '--module']```

**Comment by djtecha** [comment]
Yea, this is busted for me too. Tried with a different module just to make sure and it also fails

```
  hooks:
          #- id: terraform_tflint
          #args: ['--aws-region=us-west-2']     
          #    - id: terraform_fmt
    - id: terraform_docs
      args: ['--dest=TEST.md']


```
Terraform docs...........................................................Failed
- hook id: terraform_docs
- exit code: 1

getopt: unrecognized option '--dest'

**Comment by grzegorzlisowski** [comment]
The story is the same for both terraform_docs and terraform_tflint as they use the same method of sorting out arguments (getopt) which is making handling arguments in specific way.

**Comment by antonbabenko** [comment]
As @Vlaaaaaaad is saying but enquote values to be like strings. Like this:

```
    - id: terraform_docs
      args: ["--args=--sort-by-required", "--args=--with-aggregate-type-defaults", "--args=--no-escape"]
```

**Comment by matsest** [comment]
OK, that seems to be working. Would be nice for it to be consistent with other pre-commit hook args (as the README indicates).

**Comment by antonbabenko** [comment]
Agree, we can update README to show the examples for both.

**Comment by antonbabenko** [comment]
[v1.25.0](https://github.com/antonbabenko/pre-commit-terraform/releases/v1.25.0) of `pre-commit-terraform` has been just released after #77 was merged which means that `terraform_tflint` is usable.

Closing this issue.
